src/ssl/ngx_lua_kong_ssl.c

```diff
@@ -127,6 +127,65 @@ ngx_lua_kong_ssl_disable_session_reuse(ngx_connection_t *c)
 }
 
 
+int
+ngx_lua_kong_ssl_get_upstream_certificate(ngx_connection_t *c,
```

Comment:
We can reuse most of `ngx_lua_kong_ssl_get_full_client_certificate_chain()`. How about adding a function, say `get_full_peer_certificate_chain`, and then modifying both `ngx_lua_kong_ssl_get_full_client_certificate_chain` and `ngx_lua_kong_ssl_get_full_upstream_certificate_chain` to call it?